src/vnsw/opencontrail-vrouter-netns/opencontrail_vrouter_netns/vrouter_hyperv.py

```diff
@@ -0,0 +1,469 @@
+import os
+import time
+import sys
+import argparse
+import subprocess
+import uuid
+
+import netaddr
+import requests
+import json
+import paramiko
+
+
+def validate_uuid(val):
+    try:
+        if str(uuid.UUID(val)) == val:
+            return val
+    except (TypeError, ValueError, AttributeError):
+        raise ValueError('Invalid UUID format')
```

Comment:
This is duplicate code from vrouter_netns.py but we can't import that module on Windows. Should we consider refactoring both vrouter_netns and this?